core/ir/aarch64/instr_create_api.h

```diff
@@ -108,20 +108,6 @@
 /** Create an operand specifying LSL, the default shift type when there is no shift. */
 #define OPND_CREATE_LSL() opnd_add_flags(OPND_CREATE_INT(DR_SHIFT_LSL), DR_OPND_IS_SHIFT)
 
-/** INSTR_CREATE_sys() operations are specified by the following immediates. */
-enum {
-    DR_DC_ZVA = 0x1ba1,    /**< Zero dcache by address. */
```

Comment:
I guess this was added after the last official release so this removal does not need to be mentioned in the changelog as a compatibility break.

Reply:
Yes.